lib/unleash/toggle_fetcher.rb

```diff
@@ -4,20 +4,26 @@
 
 module Unleash
   class ToggleFetcher
-    attr_accessor :toggle_cache, :toggle_lock, :toggle_resource, :etag, :retry_count
+    attr_accessor :toggle_cache, :toggle_lock, :toggle_resource, :etag, :retry_count, :bootstrapper
 
     def initialize
       self.etag = nil
       self.toggle_cache = nil
       self.toggle_lock = Mutex.new
       self.toggle_resource = ConditionVariable.new
       self.retry_count = 0
+      self.bootstrapper = Unleash.configuration.bootstrapper
 
       # start by fetching synchronously, and failing back to reading the backup file.
       begin
-        fetch
+        if !self.bootstrapper.nil? # if the consumer provides a bootstrapper, we're going to assume they want to use it
```

Comment:
I am unsure if I understand this code block. We want to:
1. Load configuration from the boostrapper 
2. Load configuration from the Unleash API (fetch) as soon as possible.

Reply:
Is it ok to wait for the next update cycle? Or should that happen immediately after loading via the bootstraping mechanism?